package.json

```diff
@@ -19,7 +19,7 @@
     "http-status-codes": "^2.1.4",
     "launchdarkly-js-client-sdk": "^2.20.0",
     "nuxt": "^2.15.3",
-    "sbc-common-components": "2.4.40"
+    "sbc-common-components": "2.4.45"
```

Comment:
Use the latest (46)...

https://github.com/bcgov/sbc-common-components/releases

Reply:
oh sorry, updated!

Reply:
Sorry, I don't have permission to resolve this conversation, but it is resolved.